jsonnet/templates/vegeta-wrapper.jsonnet

```
@@ -0,0 +1,385 @@
+local grafana = import 'grafonnet-lib/grafonnet/grafana.libsonnet';
+local es = grafana.elasticsearch;
+
+// Panels
+local rps = grafana.graphPanel.new(
+  title='RPS (rate of sent requests per second)',
+  datasource='$datasource',
+  format='reqps',
+  legend_max=true,
+  legend_avg=true,
+  legend_alignAsTable=true,
+  legend_values=true,
+  transparent= true,
+)
+  {
+    yaxes: [{
+      format: 'reqps',
+      show: 'true'
+    },
+    {
+      format: 'pps',
+      show: 'false'
+    }],
+    fill: 2
+  }
+.addTarget(
+  es.target(
+    query='uuid: $uuid AND hostname: $hostname AND targets: $targets AND iteration: $iteration',
```

Hey @baerdal, this query is not working properly on my env. I had to double quote $targets and put it at the end of the query to be able to filter out by this field.
```suggestion
    query='uuid: $uuid AND hostname: $hostname AND iteration: $iteration AND targets: "$targets"',
```

Can you verify it?